x-pack/plugins/security_solution/common/search_strategy/security_solution/hosts/common/index.ts

```diff
@@ -21,6 +21,12 @@ export enum HostsFields {
   hostName = 'hostName',
 }
 
+export interface DocValueFieldsInput {
```

Comment:
I think that this type belong more in this file `x-pack/plugins/security_solution/common/search_strategy/security_solution/index.ts` because it is not generic to host but to our all queries